### ISSUE TITLE
Opening Harmony Composer on Mac misplaces (and duplicates) gfx files by using wrong path

### ISSUE THREAD

**Issue by marcos10soares** [Microchip-MPLAB-Harmony/gfx#7]
Mac OS Mojave 10.14.6

When I open harmony composer, save and try to generate code it adds gfx files in wrong directory (see below).

For example:
 - should create the file in:
/Users/USER/**HarmonyProjects**/projectName/firmware/src/config/VariantName/gfx/hal/src/gfx.c

 - creates the file in:
/Users/USER/**HarmonyProjects**/projectName/firmware/src/config/VariantName/gfx/hal/src/Users/USER/**microchipHarmony**/gfx/middleware/aria/hal/src/gfx.c

It looks like it is trying to copy the file from the Microchip Harmony folder to the project but instead it uses both paths.

OUTPUT:

```
build/VariantName/production/_ext/1680748723/gfx.o: In function `GFX_Initialize':
/Users/USER/HarmonyProjects/projectName/firmware/src/config/VariantName/gfx/hal/src/Users/USER/microchipHarmony/gfx/middleware/aria/hal/src/gfx.c:40: multiple definition of `GFX_Initialize'
build/VariantName/production/_ext/1680748723/gfx.o:/Users/USER/HarmonyProjects/projectName/firmware/src/config/VariantName/gfx/hal/src/Users/USER/microchipHarmony/gfx/middleware/aria/hal/src/gfx.c:40: first defined here
```

This error happens to all the gfx files.

How can I solve this?

**Comment by mhc-microchip** [comment]
Hi,

Unfortunately this is a known bug with with the MHC in non-Windows environments.  It can be worked around by removing all the generated GFX files from your MPLABX project before each generate.  Or, if your mhc version is up-to-date, you can try [this](https://app.box.com/s/8by82izmlwgwijxvj3avjio077659idd) patched version of MHC 3.3.5.

Thanks

**Comment by marcos10soares** [comment]
Thank you, that .jar fixed the issue!
